widgetbook_for_widgetbook/lib/navigation/navigation_bloc_provider.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:widgetbook_core/widgetbook_core.dart';
+
+class NavigationBlocProvider extends StatelessWidget {
+  const NavigationBlocProvider({
```

Comment:
I think we should talk about how we handle this. I think it's quite confusing to have these widgets.